firefox_ui_harness/options/functional.py

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import firefox_puppeteer
+import firefox_ui_tests
+
+from . import FirefoxUIOptions
+
+
+class FunctionalOptions(FirefoxUIOptions):
+
+    def parse_args(self, *args, **kwargs):
+        options, tests = FirefoxUIOptions.parse_args(self, *args, **kwargs)
+
+        tests = [firefox_puppeteer.manifest, firefox_ui_tests.manifest_functional]
```

Comment:
Just to note... when it comes to reporting the results should end-up in different buckets for functional and remote tests. If we do not have separate entry scripts and classes, there will be no way to distinct those on the treeherder side. Keep in mind that we also have an entry script for update tests, and those also overwrite possible arguments. I'm going to update the command line usage docstring and strip of the arguments for the new entry scripts.